test/e2e/framework/util.go

```diff
@@ -260,13 +261,29 @@ func log(level string, format string, args ...interface{}) {
 	fmt.Fprintf(GinkgoWriter, nowStamp()+": "+level+": "+format+"\n", args...)
 }
 
+// Used for important logs, controlled by ginkgo, on by default
 func Logf(format string, args ...interface{}) {
 	log("INFO", format, args...)
 }
 
+// Less important logs, controlled with -v
+var totaldebugs = 0
+
+// Debugf suppresses extra logging, emits certain debug logs with a usage message if there is a lot of information being
+// suppressed.
+func Debugf(format string, args ...interface{}) {
+	totaldebugs++
+	// avoids people getting lost in the sea of ginkgo/glog options.
+	if totaldebugs == 5 || totaldebugs == 100 || totaldebugs%200 == 0 {
```

Comment:
Fair enough.  I will add a log to the first line of test output that explains this if that sounds better...